frontend/src/queries/nodes/InsightViz/ComputationTimeWithRefresh.tsx

```diff
@@ -22,14 +22,17 @@ export function ComputationTimeWithRefresh({ disableRefresh }: { disableRefresh?
 
     return (
         <div className="flex items-center text-muted-alt z-10">
-            Computed {lastRefresh ? dayjs(lastRefresh).fromNow() : 'a while ago'}
+            Computed {lastRefresh ? <TZLabel time={lastRefresh} className="ml-1" /> : 'a while ago'}
             {!disableRefresh && (
-                <>
-                    <span className="px-1">•</span>
-                    <Link disabledReason={getInsightRefreshButtonDisabledReason()} onClick={() => loadData(true)}>
-                        Refresh
-                    </Link>
-                </>
+                <LemonButton
+                    className="ml-2"
+                    onClick={() => loadData(true)}
+                    disabledReason={getInsightRefreshButtonDisabledReason()}
+                    type="secondary"
+                    size="xsmall"
+                >
+                    Refresh
+                </LemonButton>
```

Comment:
This one change I think we should not go through with in the current UI, because it makes this area of the UI _very_ button-y visually – the 3000 primary/secondary buttons have a lot of weight to them. Might be good to revisit if we redesign the query builder

Reply:
I tend to agree... I'm also wondering, since most of the time the refresh info is alone in this line, if we might just place it in the line above with the other options and save space. But that can be done later.